### ISSUE TITLE
"Reason for Separation" Breakdown

### ISSUE THREAD

**Issue by stecklow** [ayyubibrahimi/us-post-data#14]
States **with** reason for an officer's separation:

- Florida
- Georgia
- Iowa (it's recorded as basically an employment change, so it includes promotions as well)
- Idaho
- Illinois
- Indiana
- Kentucky
- Maryland (it seems like it, but I'm not 100% sure — can discuss with Julie/MDOPD)
- North Carolina
- North Dakota
- New Mexico
- Oregon
- Tennessee
- Utah
- Vermont
- Washington
- West Virginia
- Wyoming

States with**out** the reason for an officer's separation (as of right now):

- Alaska (unclear why this wasn't provided, it can be sought in a new request)
- Arizona (earlier 2019 release included reason for separation, so a new request is pending)
- California (they probably have it, but it may take litigation to get it)
- Kansas (unlikely we'll ever get it)
- Kentucky (I feel like we could get it if we pushed)
- Mississippi (no idea if we could get it)
- North Carolina (I feel like we could if we pushed)
- Ohio (I feel like we could if we pushed)
- South Carolina (I feel like we could if we pushed)
- Texas (expressly not public)

**Comment by ayyubibrahimi** [comment]
Neither the index nor the raw tables for Alaska seem to have an end date column. Is this expected?

**Comment by stecklow** [comment]
Not really. It is probably a good idea for me to file a new request anyway, so if the data looks too messy to work with, I can prioritize that.

**Comment by ayyubibrahimi** [comment]
Sounds good. Just re-processed all of the states with separation reason data that weren't showing up as having separation reason data in the first pass. 

There are a good chunk that of states that have never been processed that have sep reason data.

**Comment by stecklow** [comment]
Let me know what I can do

**Comment by ayyubibrahimi** [comment]
Got through a few of the states during a long flight. Updated the list above. 

Couples of qs: 

- It looks like neither AK or NC have end dates, just the last "hire date". I think that this is still useful/worth displaying on the site. Thoughts? 

- Is Iowa not ocr'd? Maybe this doesn't matter because Ben already processed it. Could you direct me to that table again? 

- I don't see separation_reason data for KY. Could you point me to where you're seeing a table w separation data on dropbox? 

- NC data seems more like "certification_status" than "separation_reason". Thoughts?

**Comment by stecklow** [comment]
Hey, sorry for the delay. 

I have a new request out with AK which should hopefully give us end dates. I do think that end dates are really needed. 

I'm looking at the raw file, and it looks to me like NC does have separation date, if you isolate the ACTIONSTATUS to only "Separated" and use the ACTIONEFFECTIVEDATE as the separated date. Or have you already looked at that and it doesn't work?

**Comment by ayyubibrahimi** [comment]
Lovely. Thanks.

Sorry, Indiana, not NC, is the other state with no end date.

**Comment by stecklow** [comment]
The new data for Indiana that Mirror Indy got actually does have it: https://www.dropbox.com/scl/fi/8h3ocpaps0elrq50o1yny/Officer-Employment-History-7.25.24.xlsx?rlkey=gqv3fn4jojo548kzyn8mtd7yt&dl=0

It hasn't been processed though. Let me follow up with Emily there to see if she's had a chance to use the code that I shared with her from Justin.

**Comment by ayyubibrahimi** [comment]
This should be updated. I believe there's new data with reason for sep data.

**Comment by ayyubibrahimi** [comment]
I was talking more generally than any specific state. CA, for example, is still listed as providing data pending some litigation. Just assumed this entire chain of comms is out of date.
